pkg/apis/cluster/mutation/mutation.go

```diff
@@ -30,11 +31,23 @@ func MutateClusterTaints(taints []corev1.Taint) {
 	}
 }
 
+// MutateClusterResourceModels set cluster resource models for given order and boundary value.
+func MutateClusterResourceModels(models []clusterapis.ResourceModel) {
+	sort.Slice(models, func(i, j int) bool {
+		return models[i].Grade < models[j].Grade
+	})
+
+	for index := range models[0].Ranges {
+		models[0].Ranges[index].Min.Set(0)
+	}
+
+	for index := range models[len(models)-1].Ranges {
+		models[len(models)-1].Ranges[index].Max.Set(math.MaxInt64)
+	}
+}
```

Comment:
Can you add a unit test for it?
My concerns are:
1. Can the caller receives the sorted result?
2. We can ensure that the len(models) always >=1, right?

By the way, I think we can have more precise name, like `StandardizeClusterResourceModels`.

Reply:
Sort actually edit the subscript element of the slice. In this case, the caller will receive the sorted result but If you assign a value directly to the slice, it will not work.